tests/dummy/app/styles/app.scss

```diff
@@ -0,0 +1,187 @@
+@import "g-reset";
```

Comment:
I saw in https://github.com/lblod/ember-rdfa-editor-citaten-plugin/pull/21/files that you can now just use
`@import 'ember-rdfa-editor/a-dummy';`